arm-network/2017-03-01/swagger/virtualNetworkGateway.json

```diff
@@ -1170,12 +1173,15 @@
         },
         "tier": {
           "type": "string",
-          "description": "Gateway SKU tier. Possible values are: 'Basic', 'HighPerformance','Standard', and 'UltraPerformance'.",
+          "description": "Gateway SKU tier. Possible values are: 'Basic', 'HighPerformance','Standard', 'UltraPerformance', 'VpnGw1', 'VpnGw2' and 'VpnGw3'.",
```

Comment:
Hi I would recommend, removing the possible values in the description and it would be auto generated from the documentation tools